### PR TITLE
app-i18n/poedit: Version bump 3.0.1, enable optional features.

### PR DESCRIPTION
**Pull request by tastytea**
**Changes:**
  - Version bump ~~2.4.3~~ 3.0
  - Added optional dependencies:
    - sci-libs/cld2¹
    - dev-cpp/cpprest
  - Enabled support for optional dependencies.
  - Dropped all arches except ~amd64 because of the dependencies.

¹ I should point out that [cld2](https://github.com/CLD2Owners/cld2) seems to be abandoned since august 2015.

### CODE REVIEW

File: app-i18n/poedit/poedit-2.4.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+	l10n_find_plocales_changes 'locales' '' '.mo'
+	l10n_for_each_disabled_locale_do my_rm_loc
+
+	setup-wxwidgets
+	xdg_src_prepare
+}
+
+src_configure() {
+	# CLD2 and C++ Rest are not available in Gentoo, bug 674916
```
Comment (00-matt) [inline app-i18n/poedit/poedit-2.4.3.ebuild:53]: Outdated comment?

File: app-i18n/poedit/poedit-2.4.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	local myeconfargs=(
+		$(use_with cld2)
+		$(use_with cld2 cpprest)
```
Comment (00-matt) [inline app-i18n/poedit/poedit-2.4.3.ebuild:56]: Should this be `$(use_with crowdin cpprest)`?

File: app-i18n/poedit/poedit-2.4.3.ebuild (hunk 1 above)
Comment (tastytea) [reply app-i18n/poedit/poedit-2.4.3.ebuild]: Yeah, fixed.

File: app-i18n/poedit/poedit-2.4.3.ebuild (hunk 2 above)
Comment (tastytea) [reply app-i18n/poedit/poedit-2.4.3.ebuild]: Oopsie! Fixed.

**Comment by tastytea** [comment]
> Let's maybe dump cld2?

cld2 is used for language detection. When poedit is compiled without it, the source language [is assumed to be english](https://github.com/vslavik/poedit/commit/a1bdc6f6b266d60b4ecd2c33fd17f58294336e21). If the source language is not english, translation suggestions don't work. That is a pretty important feature, so I would rather keep it.

**Comment by tastytea** [comment]
Rebased on master, bumped EAPI to 8 in all 3 ebuilds, removed unused gnome2-utils eclass in poedit.

**Comment by hansfn** [comment]
Update PR for 3.0.1 which is newly released?

Anything stopping the current 3.0.0 from being merged? I see that a review was requested from @leio 2 months ago ...

**Comment by rahilarious** [comment]
Any progress on this?

**Comment by tastytea** [comment]
> Any progress on this?

current version is [3.4.2](https://github.com/vslavik/poedit/releases/tag/v3.4.2-oss), but i'm not interested anymore. feel free to copy this and make your own PR.

**Pull request closed without merging**